Resources/Prototypes/_LostParadise/Entities/Objects/Devices/pda.yml

```
@@ -83,4 +83,69 @@
+      - SecWatchCartridge
+
+# Delta Squad
+
+- type: entity
+  parent: BasePDA
+  id: LPPDeltaCommanderPDA
+  name: Delta Commander PDA
+  suffix: Commander
+  description: Red for firepower.
+  components:
+  - type: Pda
```

yamllint (1.35.1)

[error] wrong indentation: expected 4 but found 2

(indentation)